polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaSource.scala

```diff
@@ -101,7 +101,7 @@ class ScalaSource[Interpreter <: ScalaInterpreter](val interpreter: Interpreter)
           val pos = t.pos.withStart(t.pos.end)
           (Some(accessorName), trees.dropRight(1) :+
             global.ValDef(global.Modifiers(), accessorName, tpt, global.Ident(name).setPos(pos)).setPos(pos))
-        case expr: global.TermTree =>
+        case expr if expr.isTerm =>
```

Comment:
I think the effect of this will be the same, but it looks nicer this way. Thanks!

Reply:
Thanks for the review! 

I made this change because I was running into issues evaluating a cell like `Seq(1,2,3).toList`, as it would just sit there with no output (and no `resCell#` binding either). 

Apparently that expression is not parsed into a `TermTree` but rather a `Select`, so it wasn't hitting this case and thus wasn't defining a new value. 

Changing it to use `isTerm` fixed the problem because it's true for all the reasons a `Tree` might be a term aside from being a `TermTree`, at least according to its docstring. 

https://github.com/scala/scala/blob/2.13.x/src/reflect/scala/reflect/internal/Trees.scala#L77-L84

Reply:
😲 interesting! Good thing to know... I'd thought `Select` et al were `TermTree`s, but they're not. Good catch!